### PR TITLE
Creating Example Workflow

### PR DESCRIPTION
**Pull request by retireme-bit**
Created example workflow for public use [DVOP-2133](https://zimperium.atlassian.net/browse/DVOP-2133)

### CODE REVIEW

**Comment by retireme-bit** [comment]
Tested here: https://github.com/Zimperium/zScanExampleApp/pull/6

File: workflows/zScanAction.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        apk-path: ${{ steps.apk-path.outputs.path }}
+
+    - name: Move Artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: my-artifact
+        path: ${{ steps.apk-path.outputs.path }}
+        if-no-files-found: error # 'warn' or 'ignore' are also available, defaults to `warn`
+  
+  scan:
```
Comment (samuelmontesz) [inline workflows/zScanAction.yml]: remove extra whitespace

File: workflows/zScanAction.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      uses: actions/download-artifact@v2
+      with:
+        name: my-artifact
+
+    - name: Zimperium
+      uses: zimperium/zscanmarketplace@master
+      timeout-minutes: 60
+      with:
+        # REPLACE: Zimperium Client Environment Name
+        client_env: mtddev
```
Comment (samuelmontesz) [inline workflows/zScanAction.yml]: remove extra whitespace

File: workflows/zScanAction.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Download application
+      uses: actions/download-artifact@v2
+      with:
+        name: my-artifact
+
+    - name: Zimperium
+      uses: zimperium/zscanmarketplace@master
```
Comment (danielwoodz) [inline workflows/zScanAction.yml]: Standard practice with actions is to pin to a version, not master.

**Review by samuelmontesz — APPROVED** [review]

**Pull request merged**